### DISCUSSION TITLE
Welcome to jira Discussions!

### DISCUSSION THREAD

**Discussion by ssbarnea** [pycontribs/jira#1033] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ovidiu-munteanu** [comment]
Hi, I've just noticed that version 3.0.1 was just released! That's great, excellent work!

I was wondering, is there a changelog somewhere though, especially if there are breaking changes... thanks! 😅

**Reply by adehad** [reply]
An automated changelog since the last 2.0.0 release can be found in:

1. https://github.com/pycontribs/jira/releases/tag/3.0 
2. https://github.com/pycontribs/jira/releases/tag/3.0.1

However, in terms of breaking changes, this may be hard to answer definitively. Some are obvious, like dropping of Python 2.7 and 3.5 support. But we also ported the CI system to test on a _self-hosted_ Jira Instance (i.e. Jira Server) #1017 - so if you are using Jira Cloud it may be hard to say with confidence whether there are any breaking changes. There are definitely a fair amount of bug fixes however !

**Comment by RichaLiss** [comment]
I wonder if this library can be uesd to manipulate old version of JIRA like 4.0.2
